backend/README.md

```diff
@@ -90,20 +90,44 @@ features will be missing. It's recommended to get and configure this API key if
 you plan to contribute regularly to the project, to make your environment
 closer to the production one.
 
+#### Requirements
+
+1. a Google account
+2. to be connected while following the procedure
+3. to have at least one YouTube video in the database
+
 #### Procedure
 
-* Go to https://console.cloud.google.com/apis/ and create a new project
-  named `tournesol`
+Go to https://console.cloud.google.com/apis/ and create a new project. You
+can choose the name you prefer, we suggest `tournesol`. You should now
+be automatically redirected to the project dashboard.
+
+Go to the credentials page, accessible from the menu, and create new
+`API key `credentials. Do not apply any application nor API restriction. You
+should now be able to see the API key value by clicking on it.
+
+Now configure the `YOUTUBE_API_KEY` setting with the API key value in your
+`SETTINGS_FILE`.  If you are using the dev-env, the settings file is
+[backend/dev-env/settings-tournesol.yaml](./dev-env/settings-tournesol.yaml)
 
-* Setup you credentials by getting a API key. Do not restrict its use for
-  development purpose.
+Finally, you need to enable your API key. The first time the back end will try
+to get metadata from YouTube, you will see an HTTP 403 error in the back end's
+logs, with an activation link.
```

Comment:
I created new API keys but they are all activated by default. Maybe because my project is activated, or maybe because it contains at least one activated key.

I simplified the documentation so that developers are not forced to trigger a metadata refresh to activate the key, and I plan to improve it again with @adrausch if she has not already created a personal key.

Reply:
Partially fixed by https://github.com/tournesol-app/tournesol/pull/941/commits/fd83c0291e2e011c0d9030d08670661e3cc75a28.